assets/prometheus-k8s/rules.yaml

```diff
@@ -425,6 +425,9 @@ spec:
     rules:
     - expr: sum(rate(apiserver_request_count{job="apiserver"}[10m])) BY (code)
       record: code:apiserver_request_count:rate:sum
+    - expr: sum(rate(apiserver_request_count{job="apiserver",resource=~"image.*",verb!="WATCH"}[10m]))
```

Comment:
If it is intentional to only record the image resource here then that should be reflected in the name of the recording rule.

Reply:
:facepalm: you already did that. Out of curiosity, why specifically this API?

Reply:
Managed OSDv4 instances have an SLO regarding "the proportion of successful responses to API requests to the registry as measured from Prometheus"

Reply:
This query doesn't select only requests made by the registry.  Lots of other components in openshift make many calls to the API.  

What are you trying to measure here?  What you're actually measuring is "how many API requests are made to things called image" which includes many things the registry has no impact on.

Reply:
AFAIK, the goal here is to measure "the proportion of successful responses to API requests to the registry as measured from Prometheus" 
@jewzaam might be able to explain the objective with this SLI better than I can though